README.md

```diff
@@ -1,20 +1,30 @@
-Custom Keycloak Docker
+Raincatcher Demo Keycloak example. 
 ----------------------
 
-the contents of this repository are used to create a Keycloak standalone server
-docker image built on the alpine base to provide a small image to run tests
-against. The image size is ~309 MB as opposed to the JBoss/Keycloak image which
```

Comment:
Image size is now 344 MB, but saying its "approximately half the size" might be better than having to change a value after every change.